### PR TITLE
Translated Descriptions

### LINKED ISSUE #1: Translate Descriptions
- [x] Short Description
- [x] Description  
  in file task_description.html:
  - [x] change hyperlinks to wikipedia to their german counter parts or get rid of them
    - [x] line 27
    - [x] line 60

CheckIO-to-German/Roadmap#23

### PR DESCRIPTION
**Pull request by DahliaSR**
@GiForce: Hey, pls have a look and let me know what you think about it.

Closes #1

### CODE REVIEW

File: translations/de/info/task_description.html [hunk 1]
```diff
@@ -1,66 +1,78 @@
...
 
 <p>
-    Roman numerals come from the ancient Roman numbering system.
-    They are based on specific letters of the alphabet which are combined to signify the sum
-    (or, in some cases, the difference) of their values. The first ten Roman
-    numerals are:
+    Die römische Zahlen stammen aus dem alten römischen Zahlensystem.
+    Sie basieren auf bestimmten Buchstaben des Alphabets.
+    Addiert man die Werte der römischen Ziffern, ergibt die Summe (Oder,
+    in einigen Fällen die Differenz) die Zahl.
```
Comment (GiForce) [inline translations/de/info/task_description.html:9]: Obviously you like commas 😱I personally think such small one sentence additions flow much better without. They aren't embedded clauses, parentheses even replace commas. Anyway, one thing for sure, lowercase **oder,** and why not the second comma **in einigen Fällen, die Differenz** then ?  Duden online agrees with either way. So do I 😁 One of the rare cases I'm not wrong without commas.

File: translations/de/info/task_description.html [hunk 2]
```diff
@@ -1,66 +1,78 @@
...
+</p>
+<p>Die ersten zehn römischen Zahlen lauten:
+    I, II, III, IV, V, VI, VII, VIII, IX, und X.
 </p>
-<p>I, II, III, IV, V, VI, VII, VIII, IX, and X.</p>
 <p>
-    The Roman numeral system is decimal based but not directly positional and does not
-    include a zero. Roman numerals are based on combinations of these seven symbols:
+    Das römische Zahlensystem basiert auf den Zehnerpotenzen als Basis und den
+    fünffachen Halbbasiswerten. Abgesehen von der Subtraktionsregel ist der Wert
```
Comment (GiForce) [inline translations/de/info/task_description.html]: _fünffachen Halbbasiswerten_ you meant **fünffachen Hilfsbasiswerten**,  didn't you. Or more general and simple: "auf einem Zahlensystem mit den Basiszahlen 5 und 10." Easy,  we have it all from the same Wikipedia entry :)

**Review by GiForce — CHANGES_REQUESTED** [review]

File: translations/de/info/task_description.html (hunk 1 above)
Comment (DahliaSR) [reply translations/de/info/task_description.html:9]: I **do not** like commas, I just follow the recommendations of Duden. :smile: 
:+1:

File: translations/de/info/task_description.html (hunk 2 above)
Comment (DahliaSR) [reply translations/de/info/task_description.html]: It really sounds bad
I like your version and will change it to:
"Das römische Zahlensystem basiert auf den Basiszahlen 5 und 10."
:+1:

**Review by GiForce — APPROVED** [review]

**Pull request merged**